packages/map-template/README.md

```diff
@@ -1,88 +1,114 @@
 # Map Template
 
-This app is built on React, with our Web Components used where applicable.
+# Web component: Installation and usage
 
-## Running the app
+## Using NPM
 
-To get the app up and running, clone this repo, install everything using [Lerna](https://lerna.js.org), and start the app:
+Install the package:
 
-```zsh
-$ git clone git@github.com:MapsPeople/web-ui.git
-$ cd web-ui && npm install && npx lerna run build
-$ cd packages/map-template && npm run start
-```
-
-Now open the app served on [http://localhost:3000/](http://localhost:3000/).
+`npm install @mapsindoors/map-template`
 
-The Map Template has a main `MapsIndoorsMap` React component. It wraps the whole app inside of it. It's used in a function in `App.jsx` which is imported into `index.jsx` where it's defined that an HTML element with `id="root"` will render the app. We show how that is done in `packages/map-template/index.html`.
+In your script:
 
-### Adding Google Maps API Keys or Mapbox Access Tokens
+```javascript
+import MapsIndoorsMap from '@mapspeople/map-template/dist/mapsindoors-webcomponent.es';
```

Comment:
@ammapspeople Is it correct that the file ends with `.es`?

Reply:
Nope. Some bundlers assume that it is a js file so the `.js` can be omitted, but we can't assume that works for everyone. I'll add the missing `.js`.